nav2_behavior_tree/CMakeLists.txt

```diff
@@ -222,6 +222,14 @@ install(DIRECTORY include/
 
 install(FILES nav2_tree_nodes.xml DESTINATION share/${PROJECT_NAME})
 
+install(FILES test/test_action_server.hpp
```

Comment:
Why is this not in the test directory's cmakelist? Also, you could just use `include_directories()` for the `test/` to just include it all